packages/ui-core/src/components/IconButton/IconButton.tsx

```diff
@@ -29,33 +29,34 @@ export interface IIconButtonProps {
     disabled?: boolean;
     /** Обработчик клика по кнопке */
     onClick?: (e: React.SyntheticEvent<EventTarget>) => void;
+    /** Дополнительный класс корневого элемента */
+    className?: string;
 }
 
 const cn = cnCreate('mfui-beta-icon-button');
-const IconButton: React.FC<IIconButtonProps> = props => {
-    const {
-        theme = 'green',
-        type = 'primary',
-        href,
-        target,
-        sizeAll = 'medium',
-        sizeWide,
-        sizeDesktop,
-        sizeTablet,
-        sizeMobile,
-        icon,
-        disabled,
-        onClick,
-    } = props;
-
+const IconButton: React.FC<IIconButtonProps> = ({
+    theme = 'green',
+    type = 'primary',
+    href,
+    target,
+    sizeAll = 'medium',
+    sizeWide,
+    sizeDesktop,
+    sizeTablet,
+    sizeMobile,
+    icon,
+    disabled,
+    onClick,
+    className,
+}) => {
     const currentTheme = React.useMemo(() => (
         (type === 'primary') && (theme === 'black') ? 'green' : theme
     ), [theme]);
 
     return (
         <Button
             classes={{
-                root: cn(''),
+                root: cn([className]),
```

Comment:
Давайте использовать что-то одно, или className, или classes. В данном случае classes, className нужно удалить.

```suggestion
                root: cn({
                    disabled,
                    'size-all': sizeAll,
                    'size-wide': sizeWide,
                    'size-desktop': sizeDesktop,
                    'size-tablet': sizeTablet,
                    'size-mobile': sizeMobile,
                }, [className]),
```